tasks/add-unique-granuleID/schemas/config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "title": "AddUniqueGranuleIDConfig",
+  "description": "Describes the configuration the add-unique-granule-id task component",
+  "type": "object",
+  "properties": {
+    "hashLength": {
+      "type": "string",
+      "description": "The maximum length of the computed granule hash"
```

Comment:
Could we add the unit here?

Reply:
e.g. "character length" or "length in bytes", not sure which it is.

Reply:
Yes!  69169fb63bd